Emby.AutoOrganize/Emby.AutoOrganize.csproj

```diff
@@ -35,8 +35,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="MediaBrowser.Naming" Version="1.1.7-beta" />
-    <PackageReference Include="mediabrowser.server.core" Version="3.6.0.43-beta" />
+    <PackageReference Include="Jellyfin.Common" Version="10.0.1" />
```

Comment:
Common is included in Controller

Reply:
"error CS0234: The type or namespace name 'Controller' does not exist in the namespace 'MediaBrowser' (are you missing an assembly reference?)"

this disappeared after I added that line...

Reply:
Never mind! :)